river/command/map.zig

```diff
@@ -73,7 +73,7 @@ pub fn map(
         // possible crash if the Mapping ArrayList is reallocated, stop any
         // currently repeating mappings.
         seat.clearRepeatingMapping();
-        try mode_mappings.append(new);
+        try mode_mappings.append(allocator, new);
```

Comment:
Here and the next changed line are what I'm talking about above, with `allocator` vs `util.gpa`

Reply:
The allocator arg exists for historical reasons, today we always just pass `util.gpa`. I'd prefer if we just use `util.gpa` directly here.

It would be nice to remove the allocator arg from the command interface entirely, but that belongs in a different patch if you feel like it.

Reply:
I Changed here to `util.gpa` and I'll remove the allocator args in an other PR